### ISSUE TITLE
Recovering shap values from interaction shap values

### ISSUE THREAD

**Issue by matthewgrimes** [shap/shap#896]
I just wanted to double-check something about how interaction values are defined because I see things like "cut in half," all over the documentation.

If I have a model and define `shap_values = ....shap_values` using a `TreeExplainer`, then additionally define `interaction_values = ...shap_interaction_values` using the same explainer, then I can recover `shap_values` for a specific feature by adding its *row* (or column) in `shap_interaction_values`, right? I don't need to double the off-diagonal elements?

In other words, for a given `variable` and collection of `variables`,
`shap_variable = \sum_{othervar in variables} interaction_{variable, othervar},` correct?

This is what I find when I run the package, but parts of the documentation refer to the interactions being cut in half, which suggests I should be doubling them?

Any clarity is greatly appreciated, thanks!
